README.md

```diff
@@ -160,7 +160,7 @@ graphviz`.
   ```
 - To only execute specific notebooks, pass them as arguments.
   ```sh
-  tox -- path/to/notebook.ipynb path/to/another-notbook.ipynb
+  tox -- <path/to/notebook>.ipynb <path/to/another-notbook>.ipynb
```

Comment:
```suggestion
  tox -- <path/to/notebook.ipynb> <path/to/another-notbook.ipynb>
```

Reply:
There''s a typo - missing `e` in notebook. I didn't check if other places are an issue too

Reply:
> 

I purposely left .ipynb out of the brackets because the file always ends with .ipnyb, doesn't it?  But maybe it's against programming style guidelines?

Reply:
It does always end in `.ipynb`, but I suspect it'd be more confusing to people how you have it. I've usually seen it like Frank's suggestion